commitserver/commit/commit.go

```diff
@@ -106,6 +108,18 @@ func (s *Service) handleCommitRequest(ctx context.Context, logCtx *log.Entry, r
 		return out, "", fmt.Errorf("failed to checkout target branch: %w", err)
 	}
 
+	logCtx.Debug("Checking out if hydrated already")
```

Comment:
Maybe just `Checking if hydrated already`.

Reply:
thank you! I applied it